chp02_forces/NOC_2_05_fluidresistance/sketch.js

```diff
@@ -29,7 +27,9 @@ function setup() {
 }
 
 function draw() {
-  background(127);
+  background(255);
+  fill(0);
+  text("click to reset",10,100);
```

Comment:
Ah ok, I totally missed that.  Ignore those text changes then - the p5.dom works fine.  Sorry about that!